File: BackendCitasMedicas/src/main/java/com/example/BackendCitasMedicas/Model/Tratamiento.java
package com.example.BackendCitasMedicas.Model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Tratamiento {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id_tratamiento;
    private String tipo;
    private String descripcion;
    private String frecuencia;
    private String duracion;

    @ManyToOne
    @JoinColumn(name = "id_consulta")
    private Consulta consulta;
}

File: FrontendCitasMedicas/app/src/main/java/com/example/frontcitasmedicas/Model/Tratamiento.kt
package com.example.frontcitasmedicas.Model

data class Tratamiento(
    val id_tratamiento: Long = 0,
    val tipo: String = "",
    val descripcion: String = "",
    val frecuencia: String = "",
    val duracion: String = "",
    val consulta: Consulta? = null
)
